easybuild/easyconfigs/t/TomoPy/TomoPy-1.11.0-foss-2020b.eb

```diff
@@ -0,0 +1,53 @@
+# Authors: Joachim Hein, Zdenek Matej, Lund University
+
+easyblock = 'PythonPackage'
+
+name = 'TomoPy'
+version = '1.11.0'
+
+github_account = 'tomopy'
+homepage = "https://tomopy.readthedocs.io/en/latest/"
+description = """
+Tomographic reconstruction creates three-dimensional views of an object by 
+combining two-dimensional images taken from multiple directions, for example, 
+this is how a CAT (computer-aided tomography) scanner generates 3D 
+views of the heart or brain.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('scikit-image', '0.18.1'),
+    ('OpenCV', '4.5.1', '-contrib'),
+    ('imkl', '2020.4.304'),
+    ('numexpr', '2.8.1'),
+    ('DXchange', '0.1.8'),
+    ('PyWavelets', '1.1.1'),
+]
+
+builddependencies = [
+    ('scikit-build', '0.11.1'),
+    ('CMake', '3.18.4'),
+    ('Ninja', '1.10.1'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
```

Comment:
Looks like there are tests. Can we run them? Also, is there a sanity check command we can try?

Reply:
Hi @Micket , thanks for the comment. Again we were following the [azure-pipelines](https://github.com/tomopy/tomopy/blob/1.11.0/azure-pipelines.yml) and I think they have tests and even a special performance benchmark included but it seems to be based on [pyctest](https://github.com/jrmadsen/pyctest) and partially maintained by [Argonne-National-Laboratory](https://github.com/Argonne-National-Laboratory). I was struggling with `pyctest` for days and I did not mange to do anything with that so we dropped it as a dependency. Contrary simple python import tests helped me to identify bugs even in binary components.

So I would suggest to stay simple and do not add complex test dependencies.